modules/virt-creating-vm-yaml-web.adoc

```diff
@@ -4,9 +4,9 @@
 
 :_content-type: PROCEDURE
 [id="virt-creating-vm-yaml-web_{context}"]
-= Pasting in a pre-configured YAML file to create a virtual machine
+= Creating a virtual machine from a YAML file
 
-Create a virtual machine by writing or pasting a YAML configuration file. A valid `example` virtual machine configuration is provided by default whenever you open the YAML edit screen.
+You create a virtual machine by writing or pasting a YAML configuration file in the {product-title} web console. A valid `example` virtual machine configuration is provided by default whenever you open the YAML edit screen.
```

Comment:
As in my other comment, I think this should be "You can create" or "Create" because of the "this is the only way there is to create a VM" implications of "you create a virtual machine by..."